Formula/tea.rb

```diff
@@ -0,0 +1,32 @@
+class Tea < Formula
+  desc "Command-line tool to interact with Gitea servers"
+  homepage "https://gitea.com/gitea/tea"
+  url "https://gitea.com/gitea/tea/archive/v0.8.0.tar.gz"
+  sha256 "6c73c0a7b66cdfd1e5a302257d54df06a3a41eb9bdbfeb547966db431ae23b23"
+  license "MIT"
+  head "https://gitea.com/gitea/tea.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X main.Version=#{version}")
+    bash_completion.install "contrib/autocomplete.sh" => "tea"
+    zsh_completion.install "contrib/autocomplete.zsh" => "_tea"
+
+    system "#{bin}/tea", "shellcompletion", "fish"
+
+    if OS.mac?
+      fish_completion.install ".brew_home/Library/Application Support/fish/conf.d/tea_completion.fish" => "tea.fish"
+    else
+      fish_completion.install ".brew_home/.config/fish/conf.d/tea_completion.fish" => "tea.fish"
```

Comment:
```suggestion
      fish_completion.install "#{ENV["HOME"]}/Library/Application Support/fish/conf.d/tea_completion.fish" => "tea.fish"
    else
      fish_completion.install "#{ENV["HOME"]}/.config/fish/conf.d/tea_completion.fish" => "tea.fish"
```
`.brew_home` is an implementation detail that we don't want to rely on.

Reply:
Done, but the first line is flagged as too long (120 characters), and I have no idea how I'm supposed to break it into two lines. I tried a couple things, but `brew style` failed on all of them.

Reply:
Don't mean to nag, but I believe this is the last issue blocking the PR, so I'd like some guidance on how to fix it.

Reply:
How much too long is it? You could just drop the rename I guess.

Reply:
It's 120 characters, longer than the limit (118) by two characters.

> You could just drop the rename I guess.

Can you clarify?

Reply:
Nevermind, I got what you mean, but I just tested, and Fish depends on the basename of the file (`tea` in this case) being the same as the command name, otherwise the file won't be loaded.

Reply:
How about just doing `home = ENV["HOME"]` before the `if` and using that?

> Otherwise I think your best option is to ask the devs to make it configurable (Which seems like a better solution either way)

Agreed, but we have to make do for now.